src/proto/mgmt/mgmt.proto

```diff
@@ -33,8 +33,9 @@ import "acl.proto"; // ACL-related requests
 // Generic request indicating which ranks to operate on.
 // Used in gRPC fanout to operate on hosts with multiple ranks.
 message RanksReq {
-	repeated uint32 ranks = 1;
-	bool force = 3;
+	// repeated uint32 ranks = 1;
```

Comment:
sure, I was under the impression that we shouldn't be reusing IDs and it might be useful to have information about the original ID usage but I will just remove them

Reply:
We're not guaranteeing compatibility with 1.0, so I think it's safe to just keep the messages clean for now.